modular_citadel/code/modules/mentor/mentorpm.dm

```diff
@@ -65,21 +65,21 @@
 	var/show_char = CONFIG_GET(flag/mentors_mobname_only)
 	if(C.is_mentor())
 		if(is_mentor())//both are mentors
```

Comment:
Please be sure to use more dark mode friendly colors
Some sample text would be nice to see better what is changed for us mentors

Reply:
I'll tweak it on the run.